verification/rust/Cargo.toml

```
@@ -27,9 +26,9 @@ jf-rescue = { version = "0.1.0", git = "https://github.com/EspressoSystems/jelly
   "std",
 ], default-features = false }
 
-hotshot-types = { git = "https://github.com/EspressoSystems/espresso-network", default-features = false, rev = "b3ee13f" }
-hotshot-query-service = { git = "https://github.com/espressosystems/espresso-network", rev = "b3ee13f" }
-espresso-types = { git = "https://github.com/espressosystems/espresso-network", rev = "b3ee13f" }
+hotshot-types = { git = "https://github.com/EspressoSystems/espresso-network", default-features = false, rev = "8cbb43a" }
```

we could use the tag here too 

```suggestion
hotshot-types = { git = "https://github.com/EspressoSystems/espresso-network", default-features = false, tag = "20250409-dev-node-pos-preview" }
```